### PR TITLE
First merge

### PR DESCRIPTION
**Pull request by StrongCoding**
implemented docker secrets for database credentials
fixed makefile

### CODE REVIEW

File: backend/backend/urls.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+from django.contrib import admin
+from django.urls import path, include
+from . import views
+
+urlpatterns = [
+    path('admin/', admin.site.urls),
+	path('json/', views.json, name='json'),
+    path('pong/', include('pong.urls')),
```
Comment (wrottger) [inline backend/backend/urls.py:24]: formatting

File: docker/nginx/conf/backend.conf
Comment (wrottger) [inline docker/nginx/conf/backend.conf:1]: why so many nginx configs

File: docker/postgresql/Dockerfile
Comment (wrottger) [inline docker/postgresql/Dockerfile:1]: Do we really need a Dockerfile for this?

**Review by wrottger — CHANGES_REQUESTED** [review]

File: docker/nginx/conf/backend.conf
Comment (stevebalk) [reply docker/nginx/conf/backend.conf:1]: Fixed

File: backend/backend/urls.py (hunk 1 above)
Comment (stevebalk) [reply backend/backend/urls.py:24]: Fixed with #2

File: docker/nginx/conf/backend.conf
Comment (stevebalk) [reply docker/nginx/conf/backend.conf:1]: We only use two nginx config. The configs in conf_original are only for reference.

File: docker/postgresql/Dockerfile
Comment (stevebalk) [reply docker/postgresql/Dockerfile:1]: @StrongCoding See above.

Comment (StrongCoding) [reply docker/postgresql/Dockerfile:1]: no, fixed @wrottger @stevebalk

**Review by wrottger — APPROVED** [review]
Perfection

**Pull request merged**